.github/workflows/linux_build_tests.yaml

```diff
@@ -23,23 +23,19 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
-      max-parallel: 4
```

Comment:
Why was this removed?

Reply:
I think this was a leftover. We removed custom limitations in all the other files and are currently relying on Github's default.